### ISSUE TITLE
Overriding createGlowMesh to be a flat colour instead of glowing effect?

### ISSUE THREAD

**Issue by jackkemmish** [vasturiano/three-globe#77]
Hi there,

I've been digging around the code in the repo and wondering if it's possible to override your `createGlowMesh` function to have less of a glow effect, and rather a flat colour? I'm a THREE newbie and not sure on the best approach to add my own here instead!

I am using this module as I need more control over the globe than the other `globe.gl` module you have.

Thanks,
Jack

**Comment by vasturiano** [comment]
@jackkemmish if you need more custom control over the glow mesh, perhaps the easiest is if you deactivate the one bundled with the module (via `.showAtmosphere = false`) and add your own mesh, which you can configure the way you prefer.

**Comment by jackkemmish** [comment]
Hi @vasturiano,

Do you by any chance have an example of adding a custom mesh? I see the `customLayer` you have here [https://github.com/vasturiano/three-globe#custom-layer](https://github.com/vasturiano/three-globe#custom-layer). Would it be added here for example?

Thanks,
Jack

**Comment by vasturiano** [comment]
Yes, that would work. Or you could add it to your scene directly.
